packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -328,52 +327,30 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
     }
   }
 
-  // Validated that credential is user's above
-  await prisma.credential.delete({
-    where: {
-      id: id,
-    },
-  });
```

Comment:
While valid while here because we still had the credential data in memory. I felt better moving this to the end.